config/manifests/dev-manifest.yml

```diff
@@ -23,6 +23,7 @@ applications:
     ENV: $HOME/.profile
     GET_AN_IDENTITY_WEBHOOK_SECRET: ((GET_AN_IDENTITY_WEBHOOK_SECRET))
     GOOGLE_ANALYTICS_ID: "G-4QYNT5ZME8"
+    GOOGLE_BQ_SERVICE_ACCOUNT_BASE64: ((GOOGLE_BQ_SERVICE_ACCOUNT_BASE64))
```

Comment:
I think manifests are for the old deployments before azure and not the new ones, not sure we need to amend anything here